static/styles/rendered_markdown.css

```diff
@@ -309,8 +309,15 @@
         margin-bottom: 5px;
         margin-left: 5px;
         height: 100px;
-        display: block !important;
+        width: 150px;
+        display: inline-block !important;
```

Comment:
We need an explicit `width` on the containers to make sure our layout doesn't jump around as images are loaded! The containers also need an `inline-block` display so they don't take up an entire row on their own; only what they need.

Reply:
Can you move this information into code comments? Seems valuable for posterity to put it there.